storagenode/reputation/service.go

```diff
@@ -41,6 +41,23 @@ func (s *Service) Store(ctx context.Context, stats Stats, satelliteID storj.Node
 		return err
 	}
 
+	report := []zap.Field{
+		zap.Stringer("Satellite ID", satelliteID),
+		zap.Int64("Successful Audits", stats.Audit.SuccessCount),
```

Comment:
non-blocking: Since we are adding the successful audit counts should we add the total audit counts so it gives us an idea of the number of failed audits, if any?

```suggestion
		zap.Int64("Total Audits", stats.Audit.TotalCount),
		zap.Int64("Successful Audits", stats.Audit.SuccessCount),
```

Reply:
Regarding audits I have two goals: (1) being able to estimate the progress of vetting, (2) recognizing problems that could lead to disqualification. (1) is served by _successful audits_, (2) is served by delta scores. I didn't initially add total audits simply because I didn't have use for it, but given you see it helpful, let's go for it.